app/sidekiq/downstream_live_job.rb

```diff
@@ -83,15 +83,27 @@ def assign_attributes(attributes)
   end
 
   def enqueue_dependencies
-    DependencyResolutionJob.perform_async(
-      "content_store" => "Adapters::ContentStore",
-      "content_id" => content_id,
-      "locale" => locale,
-      "orphaned_content_ids" => orphaned_content_ids,
-      "source_command" => source_command,
-      "source_document_type" => edition.document_type,
-      "source_fields" => source_fields,
-    )
+    if @message_queue_event_type == "content_block"
+      HostContentUpdateJob.perform_async(
```

Comment:
Tiny thing here, but I'm wondering if we can set the job's class based on the event type, something like:

```ruby
dependency_job_klass = @message_queue_event_type == "content_block" ? HostContentUpdateJob : DependencyResolutionJob
dependency_job_klass.perform_async(
  "content_store" => "Adapters::ContentStore",
  "content_id" => content_id,
  "locale" => locale,
  "orphaned_content_ids" => orphaned_content_ids,
  "source_command" => source_command,
  "source_document_type" => edition.document_type,
  "source_fields" => source_fields,
)
```